app/models/portfolio_item.rb

```diff
@@ -10,6 +10,11 @@ class PortfolioItem < ApplicationRecord
 
   default_scope -> { kept.order(Arel.sql('LOWER(portfolio_items.name)')) }
 
+  after_create    { update_portfolio_stats }
+  after_discard   { update_portfolio_stats }
+  after_undiscard { update_portfolio_stats }
+  after_destroy   { update_portfolio_stats }
```

Comment:
This `after_destroy` may not be needed since we discard these objects.  The `update_portfolio_stats` method is a bit of future-proofing if more logic needs to be added to the private method.